docker/sds-api/spine-directory-service/sds/request/content_type_validator.py

```diff
@@ -0,0 +1,21 @@
+import tornado.web
+from tornado.httputil import HTTPHeaders
+
+from request.http_headers import HttpHeaders
+from utilities import integration_adaptors_logger as log
+
+logger = log.IntegrationAdaptorsLogger(__name__)
+valid_accept_types = ['application/json', 'application/fhir+json', 'application/fhir+xml']
+
+
+def get_valid_accept_type(headers: HTTPHeaders):
+    accept_type = headers.get(HttpHeaders.ACCEPT, 'application/json')
+    if accept_type == '*/*':
```

Comment:
That's the default for curl if Accept header is missing

Reply:
Never too old to learn! :)